### PR TITLE
Torch SVF Filters

### PR DESCRIPTION
**Pull request by jorshi**
State variable filter implementation that seems to be working in Torch and computing gradients that can be used to optimize it.

### CODE REVIEW

File: src/ddspdrum/torchmodule.py [hunk 1]
```diff
@@ -485,3 +485,189 @@ def _forward(self, control_in: T, audio_in: T) -> T:
...
+            [
+                TorchParameter(
+                    value=cutoff,
+                    parameter_range=ParameterRange(5.0, nyquist, "log"),
+                    parameter_name="cutoff"
+                ),
+                TorchParameter(
+                    value=resonance,
+                    parameter_range=ParameterRange(0.01, 1000.0, "log"),
+                    parameter_name="resonance"
```
Comment (maxsolomonhenry) [inline src/ddspdrum/torchmodule.py:536]: Should this lower limit be 0.5 ?

**Comment by turian** [comment]
okay I just think we should be avoiding modules that are known to be slow.

File: src/ddspdrum/torchmodule.py (hunk 1 above)
Comment (jorshi) [reply src/ddspdrum/torchmodule.py:536]: So it really just needs to be greater than 0 -- I set it to 0.5 kind of arbitrarily in the numpy version -- 0.01 gives it quite a bit more room for less intense resonance and wider bandpasses / bandstops

**Comment by jorshi** [comment]
@turian -- ya I understand that. This kind of filter is important to synthesis though so I think there is an argument to including it with the idea that maybe there is a way to make it faster at some point? Going to merge this and if we decide that we don't want / never use it we can pull it out.

**Pull request merged**